lib/reference.coffee

```diff
@@ -131,11 +133,20 @@ class Reference
 
   # Get the number of verses in the given book id
   @versesInBookId: (bookId) ->
-    return books[bookId].verses.reduce((a, b) -> a + b)
+    return books[bookId-1].verses.reduce (a, b) -> a + b
+
+  # Get the number of verses in the given chapter id
+  @versesInChapterId: (chapterId) ->
+    chapter = 1
+    for book in books
+      for verse in book.verses
+        return verse if chapter is chapterId
```

Comment:
Good suggestion: I used this method. Out of range returns undefined, which I'm ok with.